content/pages/about.md

```diff
@@ -17,7 +17,7 @@ by:
   science, technology, engineering and mathematics (STEM) at large
 + Arranging outreach and educational activities in computer science and 
   technology
-+ Providing supportive, women-only spaces (trans folk welcome!)
++ Providing supportive, women-only spaces (trans women are welcome!)
```

Comment:
Thanks for looking into this!

I'd actually suggest a different wording because I think this sounds a bit like trans women being mentioned as an afterthought. How do you like

```
Providing supportive, "women-only" spaces for cis & trans women and non-binary individuals
```

Bonus points if we make "women-only" a link to our FAQ.